guides/common/modules/proc_enabling-tracer-on-a-host.adoc

```diff
@@ -1,5 +1,5 @@
 [id="enabling-tracer-on-a-host_{context}"]
-= Enabling Tracer on a Host
+= Enabling tracer on a host
 
 Use this procedure to enable Tracer on {Project} and access Traces.
```

Comment:
Good question. It looks like Tracer and Traces are used as proper nouns in the text. I think they're OK capitalized because "Traces" is being used in a specialized sense, not generic.

Reply:
Changed to Tracer.